shared/src/business/entities/externalDocument/ExternalDocumentInformationFactory.ts

```diff
@@ -38,6 +39,7 @@ export class ExternalDocumentInformationFactory extends JoiValidationEntity {
   public secondarySupportingDocuments?: object[];
   public selectedCases?: string[];
   public supportingDocuments?: object[];
+  public generationType?: string;
```

Comment:
Is there anyway for this to not be an optional property?

Reply:
Done 👍🏻